src/Query/Aggregation/AggregationInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types = 1);
+
+namespace TBolier\RethinkQL\Query\Aggregation;
+
+use TBolier\RethinkQL\Response\ResponseInterface;
+
+interface AggregationInterface
```

Comment:
This interface is missing the limit and orderBy method?

Reply:
Fixed